glad/parse.py

```diff
@@ -1453,3 +1478,52 @@ def from_element(cls, element, api=None):
     def __str__(self):
         return '{self.name}@{self.version!r}'.format(self=self)
     __repr__ = __str__
+
+
+class SpecificationDocs(object):
+    URL = None
+
+    def __init__(self, api, version, profile, extensions):
+        self.api = api
+        self.version = version
+        self.profile = profile
+        self.extensions = extensions
+
+    def load(self):
+        raise NotImplementedError
+
+    def docs_for_command_name(self, name):
+        """
+        Returns the CommandDocs for the given command name or None if not found.
+        """
+        raise NotImplementedError
+
+
+class CommandDocs(object):
+    """
+    Inline code documentation for a command/function.
+    """
+
+    # Template rendering will interpret this as a custom paragraph break.
+    # If no special break logic is needed, just use '\n'.
+    BREAK = '__BREAK__'
+
+    class Param(namedtuple('Param', ['name', 'desc'])):
+        pass
+
+    def __init__(self, brief, params, description, notes, errors, see_also):
+        self.brief = brief
+        self.params = params
+        self.description = description
+        self.notes = notes
+        self.errors = errors
+        self.see_also = see_also
+
+    def __str__(self):
+        return 'DocComment(brief={!r}, ' \
```

Comment:
Class is called `CommandDocs`.

```suggestion
        return 'CommandDocs(brief={!r}, ' \
```